packages/dds/tree/src/feature-libraries/schema-index/schemaSummarizer.ts

```diff
@@ -51,23 +51,24 @@ export class SchemaSummarizer implements Summarizable {
 		});
 	}
 
-	public getAttachSummary(
+	public summarize(
```

Comment:
since this API has multiple optional parameters, and isn't public so we can actually fix it, I think we should switch to passing in an object with named properties instead of an order list of function arguments. That will make call sites cleaner and make future changes easier.